src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1OutOfServiceBedRevisionEntity.kt

```diff
@@ -50,12 +51,12 @@ enum class Cas1OutOfServiceBedRevisionType {
 }
 
 @Suppress("detekt:MagicNumber")
-enum class Cas1OutOfServiceBedRevisionChangeType(private val value: Long) {
-  START_DATE(0b0000_0001),
-  END_DATE(0b0000_0010),
-  REFERENCE_NUMBER(0b0000_0100),
-  REASON(0b0000_1000),
-  NOTES(0b0001_0000),
+enum class Cas1OutOfServiceBedRevisionChangeType(private val value: Long, val apiValue: ApiRevisionType) {
```

Comment:
would `Cas1OutOfServiceBedRevisionType` be sufficient here?

revision + change is redundant perhaps?

Reply:
`Cas1OutOfServiceBedRevisionType` is already being used for the `revisionType` field - it determines whether a revision is an initial revision or an update.